ntbs-service/Pages/Notifications/OverviewPartials/_ClinicalDetailsOverviewPartial.cshtml

```diff
@@ -0,0 +1,98 @@
+            <div class="notification-details-label"> Days from presentation to diagnosis </div>
+            <div class="empty-cell-min-height"> @((@Model.Notification.ClinicalDetails.DiagnosisDate?.Date - @Model.Notification.ClinicalDetails.PresentationDate?.Date)?.Days) </div>
+        </div>
+    </div>
+    <div class="nhsuk-grid-row notification-overview-details">
+        <div class="nhsuk-grid-column-one-third">
+            <div class="notification-details-label"> Treatment date </div>
+            <div class="empty-cell-min-height"> @Model.Notification.ClinicalDetails.TreatmentStartDate?.ToString("dd-MMM-yyyy") </div>
+        </div>
+        <div class="nhsuk-grid-column-one-third">
+            <div class="notification-details-label"> Days from diagnosis to treatment </div>
+            <div class="empty-cell-min-height"> @((@Model.Notification.ClinicalDetails.TreatmentStartDate?.Date - @Model.Notification.ClinicalDetails.DiagnosisDate?.Date)?.Days) </div>
+        </div>
+    </div>
+    <div class="nhsuk-grid-row notification-overview-details">
+        <div class="nhsuk-grid-column-one-third">
+            <div class="notification-details-label"> Postmortem diagnosis </div>
+            <div class="empty-cell-min-height"> @(@Model.Notification.ClinicalDetails.IsPostMortem ? "Yes" : "No") </div>
+        </div>
+        <div class="nhsuk-grid-column-one-third">
+            <div class="notification-details-label"> Date of death </div>
+            <div class="empty-cell-min-height"> @Model.Notification.ClinicalDetails.DeathDate?.ToString("dd-MMM-yyyy") </div>
+        </div>
+        <div class="nhsuk-grid-column-one-third">
+            <div class="notification-details-label"> HIV test offered </div>
+            <div class="empty-cell-min-height">  </div>
+        </div>
+    </div>
+    <div class="nhsuk-grid-row notification-overview-details">
+        <div class="nhsuk-grid-column-one-third">
+            <div class="notification-details-label"> Short course treatment </div>
+            <div class="empty-cell-min-height">  </div>
+        </div>
+        <div class="nhsuk-grid-column-one-third">
+            <div class="notification-details-label"> Directly observed treatment </div>
+            <div class="empty-cell-min-height">  </div>
+        </div>
+        <div class="nhsuk-grid-column-one-third">
+            <div class="notification-details-label"> MDR treatment </div>
+            <div class="empty-cell-min-height">  </div>
+        </div>
+    </div>
+    <div class="nhsuk-grid-row notification-overview-details">
+        <div class="nhsuk-grid-column-one-third">
+            <div class="notification-details-label"> BCG vaccination </div>
+            <div class="empty-cell-min-height"> @Model.Notification.ClinicalDetails.BCGVaccinationState @(@Model.Notification.ClinicalDetails.BCGVaccinationYear != null ? " - " + @Model.Notification.ClinicalDetails.BCGVaccinationYear : "") </div>

```

Comment:
this logic will also be used later for the MDR treatment, probably treading the line of the longest amount of logic to be kept in a cshtml file, let me know what you think

Reply:
Ah! Well, the comment just above addresses this too, really :) But yes, agreed that it's creeping up on comfort limits :D